prow/github/client.go

```diff
@@ -2298,15 +2299,32 @@ func (c *client) EditIssue(org, repo string, number int, issue *Issue) (*Issue,
 	return &ret, nil
 }
 
+// GetPullRequestDiff gets the diff version of a pull request.
+//
+// See https://docs.github.com/zh/rest/overview/media-types?apiVersion=2022-11-28#commits-commit-comparison-and-pull-requests
+func (c *client) GetPullRequestDiff(org, repo string, number int) ([]byte, error) {
+	durationLogger := c.log("GetPullRequestDiff", org, repo, number)
+	defer durationLogger()
+
+	_, diff, err := c.requestRaw(&request{
+		accept:    "application/vnd.github.diff",
+		method:    http.MethodGet,
+		path:      fmt.Sprintf("/repos/%s/%s/pulls/%d", org, repo, number),
+		org:       org,
+		exitCodes: []int{200},
+	})
+	return diff, err
+}
+
 // GetPullRequestPatch gets the patch version of a pull request.
 //
-// See https://developer.github.com/v3/media/#commits-commit-comparison-and-pull-requests
+// See https://docs.github.com/zh/rest/overview/media-types?apiVersion=2022-11-28#commits-commit-comparison-and-pull-requests
```

Comment:
Please make both these links point to an english version of the docs, not the chinese one